jpo-asn-j2735-2024/src/main/java/us/dot/its/jpo/asn/j2735/r2024/J2540ITIS/ITIStext.java

```diff
@@ -34,6 +34,6 @@ public ITIStext() {
   @JsonCreator
   public ITIStext(String value) {
     this();
-    this.value = value;
+    this.setValue(value);

```

Comment:
question: As I'm going through the changed files, I see that we have duplicate `ITIStext.java` classes with the only difference being the namespace. Is this due to some asn structuring, or is there merit in merging the redundant classes?

Reply:
It is a literal translation of the module structure of the asn.1.  There are modules named both ITIS and J2540ITIS that contain some types with the same names.  I think it best to preserve it as to reflect what is in the standard rather than alter it manually.